pages/design-tokens/typesetting/font-weight.md

```diff
@@ -184,7 +188,7 @@ Your context and coding style determine how you access USWDS font weight tokens
         </td>
         <td data-title="Example">
           <span>
-            $theme-heading-font-weight: <code>'heavy'</code>
+            $heading-font-weight: <code>'heavy'</code>
```

Comment:
> [!NOTE] 
> `$theme-heading-font-weight` does not exist in USWDS. I could not find a letterspacing theme setting in USWDS, so I created a dummy project setting that looks less like a USWDS theme setting. 
>
> Alternatively, we could remove the setting example entirely OR maybe there is a theme setting that would be a good fit that I missed. Curious to hear your thoughts.

Reply:
Would updating the value for one of the `$theme-font-weight-` settings be a viable example? Bumping up the normal font-weight for example

```scss
$theme-font-weight-normal: 500
```

Reply:
Good suggestion. I've updated this in [ea807f0](https://github.com/uswds/uswds-site/pull/2379/commits/ea807f06cb0e851146d962a1cec5ca74ddc7e870)